db/migrate/20231117211156_create_posts.rb

```diff
@@ -0,0 +1,20 @@
+class Post < ApplicationRecord
+  belongs_to :user, class_name: 'User', foreign_key: :author_id
+  has_many :comments
+  has_many :likes
+
+  validates :title, presence: true, length: { maximum: 250 }
+  validates :comments_counter, :likes_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+
+  after_save :update_user_post_counter
+
+  def five_most_recent_comments
+    comments.order(created_at: :desc).limit(5)
+  end
+
+  private
+
+  def update_user_post_counter
+    user.update(posts_counter: user.posts.count)
+  end
+end
```

Comment:
- [ ] Good job creating data model, however, you are note creating tables in a proper why. Inside you should a code similar to this:

```ruby
class CreatePosts < ActiveRecord::Migration[7.0]
  create_table :posts do |t|
    t.string :title
    t.text :text
    t.integer :comments_counter
    t.integer :likes_counter
    t.timestamps 
    t.references :author, foreign_key: { to_table: 'users' } 
  end
end
```

Kindly, fix it.

Reply:
Thanks for your review, this is fixed now!
![image](https://github.com/CarlosZ96/Blog-App-MCV/assets/125598459/3209e9d4-93e7-4303-8885-2ad08d4b73da)